### PR TITLE
SDK doc updates

### PR DESCRIPTION
**Pull request by katieclay**
:clipboard: Add associated issues, tickets, docs URL here.

## Overview

Updating the HC docs to accommodate HC SDK updates.

### CODE REVIEW

File: .gitbook/contents/hc/2_addsub/1_write-handler.md [hunk 1]
```diff
@@ -6,15 +6,23 @@ The first step is to write our *off-chain handler*. This handler is responsible
 
 Let's begin by allowing our handler to receive two numbers. It will perform both addition and subtraction on these numbers. If the result of the subtraction results in an underflow (i.e. if `b > a` in the expression `a-b`), the handler will respond with an underflow error. Create a `Python` file in which to copy all the following code blocks.
 
-First, we'll need to import some `Python` libraries to help with our example:
+First, we'll need to import the Hybrid Compute SDK and some `Python` libraries to help with our example:
 
 ```python
+import { HybridComputeSDK } from '@bobanetwork/aa-hc-sdk-server';
```
Comment (wsdt) [inline .gitbook/contents/hc/2_addsub/1_write-handler.md]: oh I see where the confusion is coming from now! In the docs you mixed now Python with TypeScript. 

The SDK is available in two different programming languages. 

In Python and in TypeScript. 

That's the python one: 
https://github.com/bobanetwork/aa-hc-sdk/tree/main/server-python

That's the typescript one: 
https://github.com/bobanetwork/aa-hc-sdk/tree/main/server

We should document both.

**Review by wsdt — CHANGES_REQUESTED** [review]
-

File: .gitbook/contents/hc/2_addsub/1_write-handler.md (hunk 1 above)
Comment (katieclay) [reply .gitbook/contents/hc/2_addsub/1_write-handler.md]: Made some changes to address this

File: .gitbook/contents/hc/2_addsub/1_write-handler.md [hunk 2]
```diff
@@ -6,15 +6,40 @@ The first step is to write our *off-chain handler*. This handler is responsible
...
 ```python
+sdk = HybridComputeSDK()
+
 from web3 import Web3
 from eth_abi import abi as ethabi
-from offchain_utils import gen_response, parse_req
 ```
 
-The first two, `Web3` and `eth_abi`, are established `PyPI` libraries. The third library, `offchain_utils`, was developed by the Boba Foundation. You can view the [library](https://github.com/bobanetwork/rundler-hc/blob/boba-develop/hybrid-compute/offchain/offchain_utils.py) on our Github.
+The last two, `Web3` and `eth_abi`, are established `PyPI` libraries. The third library, `offchain_utils`, was developed by the Boba Foundation. You can view the [library](https://github.com/bobanetwork/rundler-hc/blob/boba-develop/hybrid-compute/offchain/offchain_utils.py) on our Github.
```
Comment (wsdt) [inline .gitbook/contents/hc/2_addsub/1_write-handler.md]:
```suggestion
The last two, `Web3` and `eth_abi`, are established `PyPI` libraries. The SDK, was developed by the Boba Foundation. You can view the [library](https://github.com/bobanetwork/aa-hc-sdk/tree/main/sdks) on our Github.
```

File: .gitbook/contents/hc/2_addsub/1_write-handler.md [hunk 3]
```diff
@@ -6,15 +6,40 @@ The first step is to write our *off-chain handler*. This handler is responsible
...
+
+Note: if writing your handler in TypeScript, import the Hybrid Compute SDK and its utility functions like this:
+
+```typescript
+import { HybridComputeSDK } from '@bobanetwork/aa-hc-sdk-server';
+
+const sdk = new HybridComputeSDK();
```
Comment (wsdt) [inline .gitbook/contents/hc/2_addsub/1_write-handler.md]: Python + TypeScript SDK on the backend = are equal and do the same thing. They are interchangable based on the preference by the developer. 

We shouldn't mix them in one tutorial. Ideally we have something like this for "TypeScript" and "Python" when documenting the *Backend* SDK: 

image

**Review by wsdt — CHANGES_REQUESTED** [review]
-

File: .gitbook/contents/hc/2_addsub/1_write-handler.md (hunk 2 above)
Comment (katieclay) [reply .gitbook/contents/hc/2_addsub/1_write-handler.md]: Ooo good catch! Thank you!

File: .gitbook/contents/hc/2_addsub/1_write-handler.md (hunk 3 above)
Comment (katieclay) [reply .gitbook/contents/hc/2_addsub/1_write-handler.md]: I didn't realize gitbook had tab functionality! I was planning on switching when we converted to docusaurus, but I can thankfully convert it now.

**Review by mmontour1306 — APPROVED** [review]
It looks OK for the current system. Some things will change in my PR for the 0.7 version of the AA contracts, for example the version parameter passed to the off-chain handler will increase from 0.2 to 0.3 (and this requires the offchain server to use a different algorithm when generating a signature for the response). I've also renamed the "TestCounter" contract to "TestHybrid".

I'm approving this PR and I'll start a new draft for the 0.7 updates.

**Review by wsdt — APPROVED** [review]

**Pull request merged**